### PR TITLE
Migration base

### PR DESCRIPTION
**Pull request by armandosoriano**

### CODE REVIEW

File: deploy/cdk/data-seeding.ts [hunk 1]
```diff
@@ -37,6 +31,12 @@ export async function insertData(): Promise<void> {
     })
     .promise();
 
+  await cognito.adminAddUserToGroup({
+    UserPoolId: exportsConfig.aws_user_pools_id,
+    Username: 'admin@example.com',
+    GroupName: 'admin',
```
Comment (artyom-melnikov) [inline deploy/cdk/data-seeding.ts:37]: I'm not sure you can rely on cognito groups. One user can be part of many organizations and with different access type inside each

File: frontend/package.json [hunk 2]
```diff
@@ -25,9 +25,13 @@
     "@angular/router": "9.0.3",
     "@angular/service-worker": "9.0.3",
     "@material/top-app-bar": "^5.1.0",
+    "@ngx-translate/core": "^12.1.2",
```
Comment (artyom-melnikov) [inline frontend/package.json:28]: Please make fixed versions for all dependencies

File: deploy/cdk/data-seeding.ts (hunk 1 above)
Comment (armandosoriano) [reply deploy/cdk/data-seeding.ts:37]: Hmmm good point, will dig on it and cognito groups limits. Thanks for the heads up!

**Comment by YauhenPylAurea** [comment]
Your Pull Request has been selected for a project-agnostic code review by [Aurea Central Reviewers Team](https://github.com/orgs/trilogy-group/teams/aurea-central-reviewers/members).

<details><summary>More information</summary>

_This is an initiative to improve overall code quality. If your change is **urgent**, you **don't** have to wait for our review._ 

_NOTE: This review **does not** replace your team's review._

</details>

**Comment by YauhenPylAurea** [comment]
This Pull Request is already merged, or the latest changes only contain merge commit(s). Therefore, there is no need for a code review.

**Pull request merged**